inspirehep/modules/authors/forms.py

```diff
@@ -274,13 +274,26 @@ class AuthorUpdateForm(INSPIREForm):
 
     """Author update form."""
 
-    # Hidden field to hold author id information
     bai = fields.StringField(
-        widget=HiddenInput()
```

Comment:
It might be enough, but as before it was being done here - https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/authors/forms.py#L517-L524

You should check if that code then it is not used any more and we should delete it. Or probably even better you could use the previous existing code and when creating the class `AuthorUpdateForm` set the flag `is_update` and `is_review` that should do the hiding of the fields.